src/app/cdk/side-bar/modals/modal-email/modal-email.component.html

```diff
@@ -1,100 +1,302 @@
+<app-modal noSidebar="true">
+  <app-modal-header i18n="@@record.emails">Emails</app-modal-header>
   <app-modal-footer>
-    <button mat-raised-button color="primary" (click)="saveEvent()">
+    <button
+      mat-raised-button
+      color="primary"
+      (click)="saveEvent()"
+      [disabled]="!this.emailsForm.valid"
+      i18n="@@shared.saveChanges"
+      id="save-emails-button"
+    >
       Save changes
     </button>
-    <button mat-stroked-button color="primary" (click)="closeEvent()">
+    <button
+      mat-stroked-button
+      color="primary"
+      (click)="closeEvent()"
+      i18n="@@shared.cancel"
+      id="cancel-emails-button"
+    >
       Cancel
     </button>
   </app-modal-footer>
 
   <div class="container" [formGroup]="emailsForm">
     <div class="row">
       <div class="col">
-        <p>ORCID works best when you have multiple email addresses.</p>
+        <p i18n="@@side-bar.OrcidWorkBest">
+          ORCID works best when you have multiple email addresses.
+        </p>
         <p>
-          Per the ORCID <a href="/content/orcid-terms-use">terms of use</a> you
-          may only add email addresses that you have control over. Each time you
-          add an email address we will send you an email asking you to verify
-          it.
+          <ng-container i18n="@@side-bar.perOrcid">
+            Per the ORCID
+          </ng-container>
+          <a
+            target="_blank"
+            rel="noopener noreferrer"
+            href="/content/orcid-terms-use"
+            i18n="@@side-bar.termOfUse"
+            >terms of use</a
+          >
+          <ng-container i18n="@@side-bar.youMayOnly">
+            you may only add email addresses that you have control over. Each
+            time you add an email address we will send you an email asking you
+            to verify it.
+          </ng-container>
         </p>
       </div>
     </div>
     <div class="row">
-      <h3 class="col">Primary email address</h3>
+      <h3 class="col" i18n="@@side-bar.primaryEmail">Primary email address</h3>
     </div>
 
+    <!-- PRIMARY EMAIL -->
     <div class="row">
       <ng-container *ngFor="let email of emails">
         <div
           *ngIf="email.primary"
-          class="row email-input-line no-wrap"
-          [formGroupName]="email.value"
+          class="row email-input-line"
+          [formGroupName]="email.putCode"
         >
-          <div class="col l12 m8 s4">
+          <div class="col l12 m8 s4 email-input-container">
             <mat-form-field
+              *ngIf="!showEmailAsVerified(email.putCode)"
               appearance="outline"
-              class="mat-form-field-min"
-              [ngClass]="{ 'email-activated': true }"
+              class="mat-form-field-min verify-icon"
             >
-              <input #emailInput matInput formControlName="email" />
+              <input
+                matInput
+                matInput
+                formControlName="email"
```

Comment:
The placeholder is missing here

Reply:
Neither the design nor the old app use placeholders on here. 
Let's mention this to Dan the Friday. 

Marking as solve at the moment.